vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/VimCommandLine.kt

```diff
@@ -11,6 +11,8 @@ package com.maddyhome.idea.vim.api
 import javax.swing.KeyStroke
 
 public interface VimCommandLine {
+  public val caret: VimCommandLineCaret
```

Comment:
I don't understand why this change is included in this commit.

Reply:
I don't have any objective reason for this; I just felt that this field would look better at the top of the class and didn't want to create a separate commit for such a minor change